mkdocs.yml

```diff
@@ -77,6 +79,11 @@ nav:
     - Security: governance/SECURITY.md
 
 plugins:
+  - blog:
+      # NOTE: configuration options can be found at
+      # https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/
+      blog_dir: blog
+      blog_toc: true
```

Comment:
Can you please add:

```
post_excerpt: required
post_excerpt_max_authors: 2
```

This should fix the issues with only one author rendering on the excerpt.

Reply:
Done